content/intro-curriculum/Before-The-Workshop.md

```diff
@@ -3,6 +3,7 @@ author: Jeffrey W. Hollister & Luke Winslow
 date: 9998-01-01
 slug: Before
 title: Before The Workshop
+draft: True
```

Comment:
I think we should just wait for 3.4.1 to update the screenshots. I've added an issue so we don't forget, #189